src/_common/video/embed/embed.ts

```diff
@@ -39,17 +39,23 @@ export default class AppVideoEmbed extends Vue {
 		}
 
 		let url: string;
+		const queryParams = [];
 
 		if (this.videoProvider === 'youtube') {
 			url = 'https://www.youtube.com/embed/' + this.videoId;
+			queryParams.push('rel=0');
```

Comment:
actually, I changed my mind. put a comment above this with the link to information of what it does.